monsieurkayode/PSAdmin
# #161177929 Install and setup development tools

#### What does this PR do?
Add project configurations for building the application
#### Description of Task to be completed?
- Install dependencies and tools
- Setup code linting with `ESLint`
- Setup `Babel`
- Setup testing with `Jest`
- Setup `Webpack`
- Add editor configuration
- Create an `Express` server
#### How should this be manually tested?
- Clone the repo and checkout to this branch using the command `git checkout ch-setup-development-environment-161177929`
- Run `yarn` in your terminal to install dependencies
- Run `yarn dev` to start the application
- Run `yarn build:dev` to build the application
- Run `yarn start` to serve your build
- Navigate to [http://localhost:8080](http://localhost:8080) or [http://localhost:5000](http://localhost:5000) if serving your build
- You should see `PS Administrator` displayed on the page
#### What are the relevant pivotal tracker stories?
- [#161177929](https://www.pivotaltracker.com/story/show/161177929)
#### Background Context
None
#### Screenshots
![image](https://user-images.githubusercontent.com/26303683/46904402-73c12e80-cedb-11e8-8d1d-d5fcbbb2c387.png)
